dashboard/app/models/skill.rb

```diff
@@ -17,7 +17,9 @@
 class Skill < ApplicationRecord
   validates :description, presence: true
 
-  has_and_belongs_to_many :levels, join_table: 'levels_skills'
+  has_and_belongs_to_many :levels, join_table: 'levels_skills', dependent: :delete_all
```

Comment:
@Erin007 I believe this `dependent: :delete_all` means that the associated `levels_skills` will be deleted when a `skill` is deleted.  I also tested this out locally to confirm this.